app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="tab_text">25sp</dimen>
+    <dimen name="dp_50">50dp</dimen>
+    <dimen name="dp_8">8dp</dimen>
+    <dimen name="dp_5">5dp</dimen>
+    <dimen name="dp_10">10dp</dimen>
+    <dimen name="dp_16">16dp</dimen>
+    <dimen name="dp_24">24dp</dimen>
+    <dimen name="dp_22">22dp</dimen>
+    <dimen name="dp_30">30dp</dimen>
+    <dimen name="dp_52">52dp</dimen>
+    <dimen name="dp_81">81dp</dimen>
+    <dimen name="dp_85">85dp</dimen>
+    <dimen name="dp_200">200dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_200` appears to be unused
 Unused resources make applications larger and slow down builds.